src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/ShowCommand.java

```diff
@@ -58,6 +58,21 @@ public CommandResult execute(Model model) throws CommandException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, ShowCommand.MESSAGE_USAGE));
         }
 
+        switch (travelPlanObjectString) {
+        case Activity.TPO_WORD:
+            model.updateFilteredActivityList(Model.PREDICATE_SHOW_ALL_TRAVEL_PLAN);
```

Comment:
I think this can rename to SHOW_ALL? I remember I changed it to SHOW_ALL don't know when get changed back haha